core/src/main/java/io/grpc/internal/CensusTracingModule.java

```diff
@@ -272,9 +272,8 @@ public ServerStreamTracer newServerStreamTracer(String fullMethodName, Metadata
     public <ReqT, RespT> ClientCall<ReqT, RespT> interceptCall(
         MethodDescriptor<ReqT, RespT> method, CallOptions callOptions, Channel next) {
       // New RPCs on client-side inherit the tracing context from the current Context.
```

Comment:
Add a comment here about impact of using an internal/unsafe type, or when it will not be unsafe?